app/controllers/public/cart_products_controller.rb

```diff
@@ -13,4 +21,10 @@ def destroy
 
   def destroy_all
   end
+  
+  private
+  
+  def cart_product_params
+    params.require(:cart_product).permit(:product_id, :quantity, :customer_id)	
```

Comment:
`:customer_id`はcreateアクションで`current_customer.id`から直接入れ込むので不要かと思います
```suggestion
    params.require(:cart_product).permit(:product_id, :quantity)
```